ciao-4.11/contrib/bin/download_obsid_caldb

```diff
@@ -752,47 +747,43 @@ def main():
     from ciao_contrib.param_soaker import get_params
 
     # Load parameters
-    pars = get_params(toolname, "rw", sys.argv, 
-        verbose={"set":lw.set_verbosity, "cmd":verb1} )
+    pars = get_params(toolname, "rw", sys.argv,
+                      verbose={"set": lw.set_verbosity, "cmd": verb1})
 
-    infiles= pars["infile"]   
-    outdir = pars["outdir"]   
-    missing = ( "yes" == pars["missing"] )
-    background_files = ("yes" == pars["background"] ) 
-    clobber = ("yes" == pars["clobber"]) 
-        
+    infiles= pars["infile"]
+    outdir = pars["outdir"]
+    missing = "yes" == pars["missing"]
```

Comment:
I think the `()` are needed here if nothing more than just for readability.

Can maybe use `is` instead of `==` to make it more clear

Reply:
Nope, we cannot use `is` here.

The syntax works here so it's _ok_, but I think with the `()` the code is much more readable.

Reply:
I ended up moving the check into a separate function.